board/raspberrypi/rpi/rpi.c

```diff
@@ -594,3 +594,26 @@ int ft_board_setup(void *blob, struct bd_info *bd)
 
 	return 0;
 }
+
+/* TODO: Using late_init to initialize pci device with ID_RP1.
+ * RP1 pci device should be initialized by the PCI subsystem because
+ * it is under develop right now and depends from the final device-tree
+ * format from the Linux Kernel. Current device-tree format violates
+ * pci driver model. So this should be changed after upstreaming RP1
```

Comment:
It's misleading - this commit is needed for u-boot - but commit message and this comment are talking about "Linux Kernel"?

Reply:
because format was derived from the Linux Kernel. And I'm stating that we are dependent from the Linux kernel driver development state.